src/main/java/com/kodilla/ecommercee/ProductController.java

```diff
@@ -1,4 +1,34 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.domain.ProductDto;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+@RequestMapping("/v1/product")
 public class ProductController {
+
+    @GetMapping
+    public List<ProductDto> get(){
+        return new ArrayList<>();
+    }
+
+    @GetMapping({"/{id}"})
+    public ProductDto get(@PathVariable Long id){
+        return new ProductDto();
+    }
+
+    @PostMapping("/{productDto}")
```

Comment:
tak to nie przejdzie. W urlu nie mozemy w zaden sposob przeslac obiektu. Potrzebujemy @RequestBody